### DISCUSSION TITLE
Put logging into a Column?

### DISCUSSION THREAD

**Discussion by cschell** [Textualize/rich#900] [Q&A] [answered]
I love the output of `RichHandler` and would like to put it into a dedicated Column while displaying live stats in another - would that be possible with rich, or is that beyond its capabilities?

-chris

**Comment by willmcgugan** [comment, marked as answer]
You can combine Live and logging so that your stats are underneath the log messages -- which would be simple to implement. Having them side by side would be more effort as you would need to render the entire 'screen' each update.

**Comment by cschell** [comment]
Yeah, that works well enough for me, thanks for the quick reply!

For anyone trying this: It's important to explicitly use the same `console` instance for both, `Live` and the `RichHandler`, otherwise the output gets scrambled:

```python

from rich.console import Console
console = Console()

# ...

rh = RichHandler(console=console)
logging.basicConfig(handlers=[rh])

# ...

with Live(console=console, refresh_per_second=1) as live:
  # ...

```
